cmd/agent/app/run.go

```diff
@@ -81,6 +81,17 @@ func run(cmd *cobra.Command, args []string) error {
 		}
 	}()
 
+	// By default systemd redirect the stdout to journald. When journald is stopped or crashes we receive a SIGPIPE signal.
+	// Go ignore SIGPIPE signals unless it is when stdout or stdout is closed, in this case the agent is stopped. We don't want that
```

Comment:
nit: `ignores`. Also, it's not very clear what `we don't want that` refers to, maybe this would be clearer: `We never want the agent to stop upon receiving SIGPIPE, so [...]`